easybuild/easyblocks/n/nvhpc.py

```diff
@@ -135,9 +135,10 @@ def install_step(self):
         if isinstance(default_compute_capability, list):
             _before_default_compute_capability = default_compute_capability
             default_compute_capability = _before_default_compute_capability[0]
-            warning_msg = "Replaced list of compute capabilities {} ".format(_before_default_compute_capability)
-            warning_msg += "with first element of list {}".format(default_compute_capability)
-            print_warning(warning_msg)
+            if len(default_compute_capability) > 1:
```

Comment:
```suggestion
            if len(_before_default_compute_capability) > 1:
```

I have `EASYBUILD_CUDA_COMPUTE_CAPABILITIES=8.0` set and this change has no impact
```
== installing...

WARNING: Replaced list of compute capabilities ['8.0'] with first element of list 8.0

  >> running command:
```

This is because `default_compute_capability` is the string `8.0` and that has length `3`.